docs/source/conf.py

```diff
@@ -97,11 +106,7 @@
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
 #
-try:
-    import qcarchive_sphinx_theme
-    html_theme = 'qcarchive_sphinx_theme'
-except ModuleNotFoundError:
-    html_theme = 'sphinx_rtd_theme'
+html_theme = 'sphinx_rtd_theme'
```

Comment:
Just curious, fine with the choice: is the [`qcarchive_sphinx_theme`](https://github.com/MolSSI/qcarchive-sphinx-theme) broken, unmaintained, or both?

Reply:
It's functional but receives little maintenance. It was producing some weird formatting (giant font when tiny font expected), and Ben's moving qcf to plain rtd anyways.

Reply:
Ah, sounds good, no problem!